### DISCUSSION TITLE
Caddy rewrite

### DISCUSSION THREAD

**Discussion by ottobaer** [Zavy86/WikiDocs#184] [General]
Hi, 

I have been using wikidocs with caddy for a while and since it took me some trial and error to get the redirects right I wanted to share it here, maybe it will be useful for others who want to deploy with caddy. 

NB: This is only the rewrite section, not the full Caddyfile. 

```
 @rewrites not file
  route @rewrites {

    @pc path_regexp ^/(.*)$
    rewrite @pc /index.php?doc={re.1}&{file}&{query}

   }

```

**Comment by Zavy86** [comment]
+ @leomoon

**Comment by leomoon** [comment]
We also need to add the md file redirection.

**Comment by luisimasd** [comment]
I am totally new with wikidocs, and I have it also behind Caddy, is there any particular reason why I should I apply that rewrite?
It seems to work out of the box via docker.
Just asking, I am new, still learning, thanks for your patience.
